pyproject.toml

```diff
@@ -15,6 +15,8 @@ exclude = ["src/book/_build"]
 [tool.poetry.dependencies]
 python = ">=3.9,<=3.12"
 pyyaml = "^6.0.1"
+gitpython = "^3.1.43"
+coverage = "^7.5.3"
```

Comment:
Consider moving `coverage` to a `[tool.poetry.group.dev.dependencies]` group dependency (it would only be needed for development purposes, and not necessary for the audience using the package).